.github/dependabot.yml

```diff
@@ -5,6 +5,12 @@ updates:
     schedule:
       interval: "weekly"
 
+  # No ecosystem folders are watched because they roll up to the omnibus watcher
```

Comment:
The commit that introduced this change seems unrelated, can we remove it?

Reply:
Basically the bug fix in #7051 allowed us to no longer have to watch `/omnibus`. So technically, if we revert the bug fix, we should start watching that folder again.

But I don't think we should be reverting that bug fix, so I'm looking into properly fixing this bug, which seems related to symlinked folders.